src/pythie_serving/abstract_wrapper.py

```diff
@@ -90,9 +91,10 @@ def _parse_samples(request: predict_pb2.PredictRequest, model_specs: ModelSpecs)
         request_inputs = request.inputs
 
         features_names = model_specs["feature_names"]
+        missing_features = set(features_names) - set(request_inputs)
         if set(request_inputs) != set(features_names):
```

Comment:
change the condition to `if missing_features:` to avoid recomputing something on the sets

Reply:
I though about that but I realized that the client could send more features that what the model expects. In that case the `missing_features` will be empty but the `request_inputs` and `features_names` will still be different. For eg, the model expects features [a, b] and the client sends [a,b,c]